packages/@aws-cdk/aws-rds/lib/engine.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * A common interface for database engines.
+ * It has 2 sub-interfaces, IClusterEngine and IInstanceEngine.
+ * Don't implement this interface directly,
+ * instead implement one of the 2 sub-interfaces.
```

Comment:
question: is this what we do when we don't want an interface implemented? haven't had to do that yet but curious

Reply:
I mean, there is just no good reason to implement this specific interface, if you have `IClusterEngine` and `IInstanceEngine`.

Are you asking about that specific phrasing? Because I just made it up on the spot 🙂.

Reply:
yeah i guess it is the phrasing... the comment can go stale if there's a IFooEngine added... maybe just say something like


instead implement one of the sub-interfaces (i.e. [the ones we know about today] >)

Reply:
Changed the wording.